config/test_example.toml

```diff
@@ -0,0 +1,12 @@
+[subnets]
+
+[subnets.root]
+path = "/root"
+rpc_api = "https://example.org/rpc/v0"
+auth_token = "token_root"
+
+[subnets.child]
+path = "/root/f3thgjtvoi65yzdcoifgqh6utjbaod3ukidxrx34heu34d6avx6z7r5766t5jqt42a44ehzcnw3u5ehz47n42a"
```

Comment:
nit: we are currently using `f0` addresses, for subnets (although now that I see this in the future we could also consider using `f3` addresses even if they are longer because they are reorg safe, I'll track it on an issue an issue). You can use `"/root/f0100"` as an example of a valid `subnetID`

Reply:
Thanks! I'll make that change.